Sources/StreamChatUI/Components.swift

```diff
@@ -82,7 +82,41 @@ public struct _Components<ExtraData: ExtraDataTypes> {
     /// The injector used for injecting file attachment views
     public var filesAttachmentInjector: _FilesAttachmentViewInjector<ExtraData>.Type = _FilesAttachmentViewInjector<ExtraData>.self
 
-    public var channelList = ChannelList()
+    // MARK: - Channel list components
+
+    /// The logic to generate a name for the given channel.
+    public var channelNamer: ChatChannelNamer<ExtraData> = DefaultChatChannelNamer()
+
+    /// The collection view of the channel list.
+    public var channelListCollectionView: UICollectionView.Type = UICollectionView.self
```

Comment:
This is not a custom type nor it's reused.

Reply:
✅ Removed